File: TIY/scratch/Lecture24/car.js
let car = {
  color: 'red',
  type: 'Coupe',
  make: 'Chevy',
  model: 'Volt',
  speed: 0
  drive: function(speed) {
    this.speed = speed;
  }
  honk: function() {
    return 'Beep';
  }
}

console.log(car.honk());

File: Languages/Java/Unknown/Java/assignments/InheritanceMini-Activity/src/com/company/Car.java
package com.company;

public class Car {
    String make;
    String model;
    String color;

    public Car(String make, String model, String color) {
        this.make = make;
        this.model = model;
        this.color = color;
    }

    public void myOutput(){
        System.out.println("The car class output: " + make + " " + model + " " + color);
//        System.out.printf()
    }

}
